pkgs/development/python-modules/mdformat-admon/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mdformat
+, python3
+, pythonOlder
+}:
+
+let
+  python = python3.override {
```

Comment:
This is not okay inside of `pythonPackages`.

Reply:
Yep, never again ;)